src/mongocxx/test/spec/unified_tests/operations.cpp

```diff
@@ -391,12 +391,7 @@ document::value replace_one(collection& coll, client_session* session, document:
 
```

Comment:
All changes to `operations.cpp`, `client_helpers.hh` and `client_helpers.cpp` (and most to `runner.cpp`) should be gone after rebasing on [CXX-2230](https://github.com/mongodb/mongo-cxx-driver/pull/784).